common/services/prismic/index.ts

```diff
@@ -16,4 +16,4 @@ import { isString } from '@weco/common/utils/array';
 
 export const looksLikePrismicId = (
   id: string | string[] | undefined
-): boolean => (isString(id) ? /^([A-Za-z0-9-_]{9,})$/.test(id) : false);
+): id is string => (isString(id) ? /^([A-Za-z0-9-_]{9,})$/.test(id) : false);
```

Comment:
This is one of the interesting bits – it tells the type checker that if `id` returns true, it can treat it as a `string` from this point forward, which is how I was able to remove all the `as string` casting.